UNRELEASED.md

```diff
@@ -17,6 +17,9 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Enhancements
 
 - Made the `action` prop optional on `EmptyState` ([#1583](https://github.com/Shopify/polaris-react/pull/1583))
+- Added textAlign property to Button which modifies its inner text alignment. ([#1576](https://github.com/Shopify/polaris-react/pull/1576))
```

Comment:
Added `textAlign` prop to Button ([#1576](https://github.com/Shopify/polaris-react/pull/1576))

That would be enough.